pkg/controller/controller.go

```diff
@@ -65,6 +65,7 @@ func NewModifyController(
 		DeleteFunc: ctrl.deletePVC,
 	}, resyncPeriod)
 
+	informerFactory.Start(wait.NeverStop)
```

Comment:
Thoughts on doing this in `main.go`? Given that the object is created there and `main.go` relies on it to be started, feels weird to start it here.

Reply:
Moved to main ✅